integration-tests/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <quarkus.build.skip>${skipTests}</quarkus.build.skip>
+<!--        <quarkus.build.skip>${skipTests}</quarkus.build.skip>-->
```

Comment:
I think we need to get rid of this :wink:

Reply:
oops.  yeah.  usually i catch that.  ;)